Source/Editor/Surface/Archetypes/Animation.StateMachine.cs

```diff
@@ -1046,7 +1046,7 @@ public void UpdateTransitions()
                         startPos += nrm;
                         endPos += nrm;
 
-                        // Swap fo the other arrow
+                        // Swap for the other arrow
```

Comment:
This is a tricky one, maybe it should be `to`

Reply:
That's right, changed it now thanks for the hint!